### PR TITLE
Added code to print the Row wise  sum in 2d array

### PR DESCRIPTION
**Pull request by ritikakatoch**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Make Markdown Updates

**Comment by UG-SEP** [comment]
@HarshCasper please check my pr #5145

**Comment by ankitaggarwal23** [comment]
@UG-SEP Don't spam other contributors PR. This is not the right way to ask for review by commenting on other persons PR.

**Comment by UG-SEP** [comment]
> @UG-SEP Don't spam other contributors PR. This is not the right way to ask for review by commenting on other persons PR.

sorry for the that this will not from next time sorry @ankitaggarwal23

**Comment by ritikakatoch** [comment]
@HarshCasper  I have updated the readme.md.

File: Java/README.md [hunk 1]
```diff
@@ -98,6 +98,7 @@ _add list here_
 - [Maximum of Minimum for Every Window Size](cp/MaximumOfMinimumSubarrays.java)
 - [Rat In A Maze](cp/RatInAMaze.java)
 - [Median of two sorted arrays](cp/Median_of_two_sorted_arrays.java)
+- [RowWise_Sum]( Java/2d%20Array/RowWise_Sum.java)
```
Comment (HarshCasper) [inline Java/README.md]: This hyperlink is wrong.

File: Java/2d Array/RowWise_Sum.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+1 2 3 4
+9 8 7 6
+3 4 5 6
+-1 1 -10 5
+
+Output-->
+20 32 
+10 30 18 -5 
+
+ */
```
Comment (HarshCasper) [inline Java/2d Array/RowWise_Sum.java]: Keep the sample I/O at the end of the Code. And keep only one sample I/O

File: Java/2d Array/RowWise_Sum.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+Input-->
+1) First line contains the integer 't' which denotes number of test cases.
+2) First line of each test case or query contains two integer values, 'N' and 'M',
+     separated by a single space. They represent the 'rows' and 'columns' respectively,
+     for the two-dimensional array/list.
+3) Second line onwards, the next 'N' lines or rows represent the ith row values.
+
+Output-->
+1) For each test case, print the sum of every ith row elements in a single line separated by a single space.
+2) Output for every test case will be printed in a seperate line.
```
Comment (HarshCasper) [inline Java/2d Array/RowWise_Sum.java]: No need to keep the I/O format

File: Java/2d Array/RowWise_Sum.java [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+ROW WISE SUM OF A 2D ARRAY
+
+Question: For a given two-dimensional integer array/list of size (N x M), 
+find and print the sum of each of the row elements in a single line, 
+separated by a single space.
+Ex :    Let arr be an 2d array such as:
+          arr=[ [1 ,2],[3,4],[5,6],[7,8] ]
+          then rowise sum of each row in a single line seperated by space is:
+          3 7 11 15 
+i.e 1+2=3, 3+4=7, 5+6=11 and 7+8=15
```
Comment (HarshCasper) [inline Java/2d Array/RowWise_Sum.java]: No need of an example here. The sample I/O does the work

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by ankitaggarwal23 — COMMENTED** [review]
We don't accept PRs without any corresponding Issue. It is difficult for us to find your issue in tons of issues available. We can't review and merged incomplete PR's.  Closing this PR as of now. Open the new PR if you want this to be reviewed and add the proper details and do the mentioned changes requested by harsh.  
   
Capture

**Pull request closed without merging**